### DISCUSSION TITLE
Why is markdown off by default in github copilot?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#119471] [Copilot Conversations]
### Select Topic Area

Question

### Body

I noticed that github copilot doesn't work with markdown.

The reason is that it was disabled by default.
It was resolved by enabling it in settings.json.

<img width="610" alt="スクリーンショット 2024-04-14 23 22 13" src="https://github.com/community/community/assets/68766032/04e5fbab-d048-49f7-8042-8d8c21b1c6e6">

```json
{
  "github.copilot.enable": {
    "markdown": true
  }
}
```

Why this change?
Am I correct in understanding that Github Copilot is primarily intended for code completion and is not recommended for text purposes?
It was a genuine question.

**Comment by mark-spain-bayer** [comment]
I was wondering why copliot kept disabling itself whenever I started editing Markdown. Your post showed me how to fix it, but now I have the same question you have. It seems like documentation would be a prime use case for AI to speed things up so it's confusing to me that markdown would be disabled by default for copilot.

Anyway, thanks for the hint!
